### PR TITLE
added row height params and multipage table is implemented

### PR DESCRIPTION
**Pull request by zgaffarHC**

### CODE REVIEW

File: US_Cities.txt
Comment (pgaviganHC) [inline US_Cities.txt:1]: This seems like a file that should be stored with the other test artifacts.

**Comment by pgaviganHC** [comment]
Allthough the changes in this specific PR look good, I am unable to run them. This implies to me that there may have been problems with previous PRs that we will need to fix.

- There needs to be test cases to support new code. This helps document the expected functionality as well as build confidence that the code works
- There is no supporting documentation in the ReadMe to help with any new configuration. When I try running the benchmark_pipeline it failes with tex file errors. I suspect that there is some configuration steps that are needed. These need to be documented in the Readme.

Please add supporting documentaiton in the ReadMe as well as some improved test coverage to confirm that the new functionality is working. In short: there is no such thing as too many test cases!

File: US_Cities.txt
Comment (fnwaohaHC) [reply US_Cities.txt:1]: Ya this file isn't needed, when a user runs the code it will automatically be generated

**Review by fnwaohaHC — APPROVED** [review]
Testcase passed and table generation works. Everything looks good

File: US_Cities.txt
Comment (pgaviganHC) [reply US_Cities.txt:1]: In that case it should be removed from the repository and added to the .gitignore file.

**Review by pgaviganHC — APPROVED** [review]
Tests pass and I think this is OK to merge, but I do have a few comments that go beyond the scope of this PR on its own.
- I think it will be a good idea to check the repos for any test artifacts that have been generated and remove them. I also think those should be added to the .gitignore so that they stay our of the repos in the future. If test cases genrated files the test case should clean them up (delete them). I believe that this can help: https://stackoverflow.com/questions/22627659/run-code-before-and-after-each-test-in-py-test.
- Files needed for test cases should move to a folder within the tests directory. I think that includes the contents of the resources folder as well as the generated_tables folder (once moved the tests should be rerun to make sure nothing broke)
- I suspect it may be possible to remove the need to edit the PATH variable, meaning that the changes to the README may not be needed in the long run. I'm thinking along these lines: https://stackoverflow.com/questions/5971312/how-to-set-environment-variables-in-python. Previously for Tesseract I did something like this: `pytesseract.pytesseract.tesseract_cmd = os.path.join('C:/Users',getpass.getuser(),'AppData/Local/Programs/Tesseract-OCR/tesseract.exe')`
- Test coverage is a bit limited

Otherwise, for now I think this can merge. Some of these can be opened as new issues or handled with some code refactoring in upcoming branches and PRs.

**Comment by fnwaohaHC** [comment]
A quick way to buff up the testcase is:
- Rather than having the pdf files already in the repo, generate one or 2 with the code
- Then use the getter methods in class to get the filename and pass it to the pdfreader you're already using to check the length

**Comment by pgaviganHC** [comment]
> 

Yes, and all of that should be done in the test fixture.

**Pull request merged**